src/utlist.h

```diff
@@ -349,6 +349,35 @@ do {
   }                                                                                            \
 } while (0)
 
+#define LL_INSERT_INORDER(head,add,cmp)                                                        \
```

Comment:
Shouldn't there be an `LL_INSERT_INORDER2` for the benefit of lists with non-default next and prev fields?